test/ably/restchannelhistory_test.py

```diff
@@ -41,11 +42,11 @@ def test_channel_history_types(self):
         history0.publish('history2', 24.234)
         history0.publish('history3', six.u('This is a string message payload'))
         history0.publish('history4', b'This is a byte[] message payload')
-        history0.publish('history5', {'test': 'This is a JSONObject message payload'})
-        history0.publish('history6', ['This is a JSONArray message payload'])
+        history0.publish('history5', "{'test': 'This is a JSONObject message payload'}")
```

Comment:
This is not a valid test, the payload should be JSON, we support JSON-like payloads